### DISCUSSION TITLE
Multiheader (Multiindex) columns in ui.table

### DISCUSSION THREAD

**Discussion by GeorgiT-HI** [zauberzeug/nicegui#3964] [Q&A] [answered]
Greetings everyone, 

I have been looking to buld something like [this](https://codepen.io/mickey58/pen/XWWVEEN) with nicegui, but then I found this errormessage in the sourcecode: 

https://github.com/zauberzeug/nicegui/blob/7a3cfc2c9ed1f6d1532f62db061cd2451861b3b5/nicegui/elements/table.py#L124

Are Multiheaders supported? Is there any way/workaround to replicate the codepenexample from above?

Thanks
Georgi

**Comment by falkoschindler** [comment, marked as answer]
Hi @GeorgiT-HI,

As the exception says, MultiIndex columns are not supported. And as stated in the next lines, "You can convert them to strings using something like `df.columns = ["_".join(col) for col in df.columns.values]`".
See https://github.com/zauberzeug/nicegui/issues/2384#issuecomment-1895113992 for more information about the underlying problem with such columns.

**Reply by GeorgiT-HI** [reply]
Thank you, Falko!
